### ISSUE TITLE
crystal@1.13.1: hash check failed

### ISSUE THREAD

**Issue by kojix2** [neatorobito/scoop-crystal#32]
`scoop install crystal`

```
Installing 'crystal' (1.13.1) [64bit] from 'crystal-preview' bucket
crystal-1.13.1-windows-x86_64-msvc-unsupported.zip (44.2 MB) [================================================] 100%
Checking hash of crystal-1.13.1-windows-x86_64-msvc-unsupported.zip ... ERROR Hash check failed!
App:         crystal-preview/crystal
URL:         https://github.com/crystal-lang/crystal/releases/download/1.13.1/crystal-1.13.1-windows-x86_64-msvc-unsupported.zip
First bytes: 50 4B 03 04 14 00 00 00
Expected:    7c16276604bbfaf9d96a6973ea09d79168c7c227d39c9d27d6c6479baca7920f
Actual:      9c1d2a8746be249bf69d77cc56192c19b293ddb1c6a13b65c9dbd354f590efa4

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/neatorobito/scoop-crystal/issues/new?title=crystal%401.13.1%3a+hash+check+failed
```

**Comment by kojix2** [comment]
`scoop install --skip-hash-check crystal` works fine.

**Comment by neatorobito** [comment]
Hey, sorry for the radio silence here. This seems to have been resolved in 1.13.2.
